cf/field.py

```diff
@@ -13845,6 +13852,7 @@ def regridc(
         inplace=False,
         i=False,
         _compute_field_mass=None,
+        return_esmpy_regrid_operator=False,        
```

Comment:
Could this be moved to just before the  the `inplace` keyword (just some secret house style)

Reply:
Resolved in commit https://github.com/NCAS-CMS/cf-python/pull/768/commits/40b00233a50df02a3a728cbbb59c7526e31e8c9f